Manifest.toml

```diff
@@ -74,9 +74,9 @@ version = "4.0.0"
 
 [[DataStructures]]
 deps = ["InteractiveUtils", "OrderedCollections"]
```

Comment:
if the manifest changes aren't needed, please leave them out

Reply:
I was not able to run tests without `] update` - no sure why. I can revert the change though.

Reply:
It is, but in Base `\(::Matrix, ::Matrix)` is implemented in the LinearAlgebra stdlib too, so I guess this makes sens?

Reply:
Oh I didn't release this! Let me move the code into `linalg.jl` then!

Reply:
You were using `julia --project` right? The manifest should always be valid, as it's what the CI infrastructure uses. Try getting rid of it, and we'll see what CI thinks about it :slightly_smiling_face:

Reply:
I did with `--project=@.` inside the folder. I got errors about `CUstream` not defined.

Reply:
You probably didn't `instantiate` then (to have Pkg actually check out the versions in the Manifest).

Reply:
Oh I see. Thanks! I will revert this then.